lib/scala/tree/src/main/scala/edu/rpi/tw/twxplore/lib/geo/models/domain/City.scala

```diff
@@ -0,0 +1,5 @@
+package edu.rpi.tw.twxplore.lib.geo.models.domain
+
+import scala.collection.mutable.ListBuffer
+
+case class City(city: String, boroughs: ListBuffer[Borough], postcodes: ListBuffer[Postcode], state: String)
```

Comment:
missing a final, use name: String instead of city: String, then state: State. List[Borough], List[Postcode].